spec/models/user_spec.rb

```diff
@@ -114,11 +114,10 @@
 
     describe "#feed" do
       it "returns all posts from all friends of a user" do
-        user.posts << post
+        user4.posts << post
```

Comment:
Oh. I missed it in my previous feedback. I thought it's only me. Awesome you nailed it 🆒